src/app/shell/workshop-details/side-menu/contacts/contacts.component.html

```diff
@@ -3,8 +3,8 @@ <h4 class="card-title">КОНТАКТИ</h4>
   <p class="contact-info">
     <mat-icon class="card_icon">location_on</mat-icon>
     <span fxLayout="row wrap" fxLayoutAlign="start">
-      <span>{{workshop.address.city}}</span>
-      <span>{{workshop.address.street}}</span>
+      <span>{{workshop.address.city}},&nbsp;</span>
```

Comment:
please use css for line breaks preventing

Reply:
used "white-space: pre" css property to preserve whitespaces

Reply:
@dmitryy90 did I understand you correctly - I was supposed to replace "&amp;nbsp;" with css or do something else?